package.json

```diff
@@ -6,18 +6,18 @@
   "scripts": {
     "build": "sass scss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
     "postbuild": "postcss build/bitstyles.css --replace",
-    "lint:scss": "stylelint scss/**/*.scss --syntax scss",
+    "lint:scss": "stylelint --ignore-path .gitignore 'scss/**/*.scss' --customSyntax postcss-scss",
     "lint:js": "eslint --ignore-path .gitignore --ext js,jsx .",
     "lint": "yarn lint:scss && yarn lint:js",
+    "prettier": "prettier --ignore-path .gitignore --check ./scss",
```

Comment:
Is this also taking care of `.md` files? Should we add a separate `lint:md` for it?

Aside that: wouldn't it be easier for devs to handle, if we'd include prettier in the eslint config and then have `lint:js` already take care of this? Same for stylelint of course.

It's all possible with their plugin systems: https://prettier.io/docs/en/integrating-with-linters.html

Reply:
They suggest using the plugins is not ideal, and using `prettier` on the CLI directly is better. Since our previous projects use `prettier` directly, I stuck with that approach, but added prettier tasks for js, scss, and mdx. Those run on each lint task, and therefore on CI. If you think it’s better we can switch to using plugins though, wdyt?

Reply:
No strong feelings here. Fine to have them as separate tasks, as long as they're discoverable and run on CI.